transformations/character_duplication/transformation.py

```diff
@@ -0,0 +1,53 @@
+import itertools
+import random
+
+from interfaces.SentenceOperation import SentenceOperation
+from tasks.TaskTypes import TaskType
+
+
+def duplicate(text, prob=0.1, seed=42, max_outputs=1):
+    random.seed(seed)
+
+    original_text = list(text)
+    perturbed_texts = []
+    for _ in itertools.repeat(None, max_outputs):
+        perturbed_text = [
+            [letter] if random.random() > prob else [letter, letter]
+            for letter in original_text
+        ]
+        perturbed_text = [
+            letter for sublist in perturbed_text for letter in sublist
+        ]
+        perturbed_texts.append("".join(perturbed_text))
+    return perturbed_texts
+
+
+class CharacterDuplication(SentenceOperation):
+    tasks = [
+        TaskType.TEXT_CLASSIFICATION,
+        TaskType.TEXT_TO_TEXT_GENERATION,
+        TaskType.TEXT_TAGGING,
```

Comment:
How the `TaskType.TEXT_TAGGING` is relevant to this PR?

Reply:
Thank you for spotting this, you are completely right and I have removed it in eb09bbcab3d2d47742ef47e353ec315e3ab05627